### PR TITLE
update gcp

### PR DESCRIPTION
**Pull request by maryamsulemani97**
This PR updates `learn/cookbooks/gcp.md`

part of https://github.com/meilisearch/documentation/issues/1719

### CODE REVIEW

File: learn/cookbooks/gcp.md [hunk 1]
```diff
@@ -108,21 +106,21 @@ To start the configuration process, connect via SSH to your new Meilisearch inst
...
+Navigate to "Compute Engine" -> "VM instances". Click on the name of your instance, and then click on the "SSH" button (make sure to have pop-ups enabled or the window may be blocked). The connection will be established and the script will automatically run.
 
 #### 2.2.2 Option 2: Add your SSH key to the Compute Engine metadata
 
-- Make sure that you have an SSH public key (normally stored in `~/.ssh/id_rsa.pub`). If you haven't created your SSH key or want to generate a new one, you can follow [this guide](https://www.ssh.com/ssh/keygen/).
+- Make sure that you have an SSH public key (normally stored in `~/.ssh/id_rsa.pub`). If you haven't created your SSH key or want to generate a new one, you can follow [this guide](https://www.ssh.com/ssh/keygen/)
 
-- On the GCP dashboard, navigate to `Compute Engine` => `Metadata`, and click on the`SSH Keys` tab. Click on `Edit`.
+- On the GCP dashboard, navigate to "Compute Engine" -> "Metadata", and click on the"SSH Keys" tab. Click on "Edit"
```
Comment (maryamsulemani97) [inline learn/cookbooks/gcp.md:115]:
```suggestion
- On the GCP dashboard, navigate to "Compute Engine" -> "Metadata", and click on the "SSH Keys" tab. Click on "Edit"
```

File: learn/cookbooks/gcp.md [hunk 2]
```diff
@@ -80,7 +78,7 @@ Configuring your Meilisearch instance in a production environment is not just st
 
 ### 1. Make your domain name point to your instance IP
 
-If you want to use a custom domain name (or sub-domain), add an `A record` in your domain name provider account. Otherwise, you can skip this step.
+If you want to use a custom domain name (or sub-domain), add a record to your domain name provider account. Otherwise, you can skip this step.
```
Comment (dichotommy) [inline learn/cookbooks/gcp.md:81]: ⚠️ Caution, an "A record" is an actual thing! ⚠️ 

So you'll want to revert this change. [Here's a link](https://docs.gandi.net/en/domain_names/faq/record_types/a_record.html) where you can read a little more about it.

**Review by dichotommy — COMMENTED** [review]
This looks good, just one change that should be reverted 👌🏻

File: learn/cookbooks/gcp.md (hunk 2 above)
Comment (maryamsulemani97) [inline learn/cookbooks/gcp.md:81]:
```suggestion
If you want to use a custom domain name (or sub-domain), add an `A record` in your domain name provider account. Otherwise, you can skip this step.
```

**Review by dichotommy — APPROVED** [review]
LGTM!

**Comment by maryamsulemani97** [comment]
bors merge
